### PR TITLE
added documentation to the README

### PR DESCRIPTION
**Pull request by mbjones**
@jeanetteclark Please consider these documentation improvements.  Thanks!

### CODE REVIEW

**Comment by jeanetteclark** [comment]
This is a start of a conversation about the PR

**Comment by mbjones** [comment]
Ok I'll do that. Be back in a day or two with some changes.

**Comment by mbjones** [comment]
@jeanetteclark ok, its ready.

File: README.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ Here is a list:
 
 ## New documentation
 
-- We really need to say this.
\ No newline at end of file
+- We really need to say that.
```
Comment (mbjones) [inline README.md]: This really should be beta.

**Comment by jeanetteclark** [comment]
thanks @mbjones! 🚀

**Pull request merged**